.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
do we really need it? If you see some undeeded files please do not add it.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/309/commits/120b2b9689a7b719975c93904ba71db5a2ffe42e